app/components/general/AddStakeholderModal/index.tsx

```diff
@@ -209,7 +211,9 @@ function AddStakeholderModal<T extends string>(props: Props<T>) {
                     onOptionsChange={onOptionsChange}
                     name="LEAD_ORGANIZATION"
                     value={value.LEAD_ORGANIZATION}
-                    label={_ts('project.detail.stakeholders', 'leadOrganization')}
+                    label={fromAssessment
+                        ? 'Assessment Lead'
+                        : _ts('project.detail.stakeholders', 'leadOrganization')}
```

Comment:
get label from prop and then change set the default to lead organization. lets remove _ts as well. Not necessary right now